index.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html>
+  <head>
+    <link rel='stylesheet' href='style.css'>
+  </head>
+  <body>
+    <h1 id='title'>Free CodeCamp Survey Form</h1>
```

Comment:
Please, make a header tag and put the h1 tag inside the header tag. This will make your HTML code more semantic and will be helpful for SEO.